Patch 1

Saketd3769 opened the pull request:
I Saket Dhakar, Anshit Arya and Prashant Verma have worked on FPTP voting system

Saketd3769 committed:
Created FPTP

Saketd3769 committed:
Added Readme file

Saketd3769 committed:
Update Firstpastthepostvoting.sol

AthaSSiN commented:
Your code seems alright, however check out this doc

https://docs.google.com/spreadsheets/d/1lPY1fnPtlCkGDwcxQpC7W9dZVVhDgysbBPSDziLbTGY/edit?usp=sharing

FPTP was already selected by another team, and they made a PR https://github.com/pclubiitk/smart-contract-hub/pull/2 before you as well.  Along with that, you didn't fill the doc.

Try to implement another mechanism till 18th midnight if you can.

Along with that, follow the recommended directory structure. (Voting/mechanism_name)

Saketd3769 committed:
Create TwoPartyVoting.sol

Saketd3769 committed:
Create README.md

Saketd3769 committed:
Create TwoRoundVoting

Saketd3769 committed:
Create FPTP.sol

Saketd3769 committed:
Create TwoRoundVoting.sol

Saketd3769 committed:
Create README.md

Saketd3769 committed:
Delete Firstpastthepostvoting.sol

Saketd3769 committed:
Delete README.md

Saketd3769 committed:
Delete TwoPartyVoting.sol

Saketd3769 committed:
Update README.md

Saketd3769 committed:
Update README.md

Saketd3769 committed:
Delete TwoRoundVoting

Saketd3769 commented:
I have updated google docs. I also added Two round voting system. I am new to GitHub I was unable to understand the "recommended directory structure" part. I tried to correct it to. Tell me if further changes are required!
